src/routes/search/components/SearchModal.js

```diff
@@ -108,13 +108,13 @@ class SearchModal extends React.Component<PropsType, StateType> {
     const { theme, closeModal } = this.props
     const { query } = this.state
     return (
-      <Wrapper theme={theme}>
-        <SearchHeader theme={theme}
-                      query={query}
-                      closeSearchBar={closeModal}
-                      onSearchChanged={this.onSearchChanged} />
-        {this.renderContent()}
-      </Wrapper>
+        <Wrapper theme={theme}>
```

Comment:
I guess you are using an indentation of 4 spaces while we are using 2 spaces. Please change this :) If you are using IntelliJ, you can find the setting here: Settings > Editor > Code Style > JavaScript > Tabs and Indents